test/test-basic-test.jl

```diff
@@ -1,3 +1,28 @@
+using InclusiveScans
+using CUDA
+
 @testset "InclusiveScans.jl" begin
-    @test InclusiveScans.hello_world() == "Hello, World!"
+    function _cpu_inclusive_cumsum(x::Vector{Float32})
+        s = 0.0f0
+        out = similar(x)
+        for i = 1:length(x)
+            s += x[i]
+            out[i] = s
+        end
+        return out
+    end
```

Comment:
Consider using Julia's built-in functions to test against. For instance [`Base.accumulate`](https://docs.julialang.org/en/v1/base/arrays/#Base.accumulate)